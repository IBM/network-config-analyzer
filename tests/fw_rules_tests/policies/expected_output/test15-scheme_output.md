|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|connectivity_map, config: np15||||||
|||ip block: 0.0.0.0/0|[ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore]|[*]|All connections|
||[default,ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore]|[*]||ip block: 0.0.0.0/0|All connections|
||[default,ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore]|[*]|[ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore]|[*]|All connections|
||[ibm-system-new]|[(ibm-cloud-provider-lb-app in (keepalived))]|[default]|[*]|All connections|


