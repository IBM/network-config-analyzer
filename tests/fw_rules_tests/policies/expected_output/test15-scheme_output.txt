final fw rules for query: connectivity_map, config: np15:
src: 0.0.0.0/0 dst_ns: [ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore] dst_pods: [*] conn: All connections
src_ns: [default,ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
src_ns: [default,ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore] src_pods: [*] dst_ns: [ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore] dst_pods: [*] conn: All connections
src_ns: [ibm-system-new] src_pods: [ibm-cloud-provider-lb-app=keepalived] dst_ns: [default] dst_pods: [*] conn: All connections
